src/models/Player.ts

```diff
@@ -58,6 +73,27 @@ export default class Player {
         this._gameSkills.push(skill);
     }
 
+    isSkillAssessedForGame(game: Game): boolean{
+        for (const gameSkill of this._gameSkills){
+            if(gameSkill.game === game){
+                if (gameSkill.skillLevel > 0){
+                    return true;
+                }
+                break;
```

Comment:
Are you sure a "break" is correct here? Or do you want to "continue" here to continue search within the gameSkills?

Reply:
Yes "break" is intentional as the game has been found so we do not need to look further. There are two cases where we want to return false. 

1)  The game has not been found
2)  The game has not been assessed

Maybe it is more clear if i refactor it to the following:

```
for (const gameSkill of this._gameSkills){
        if(gameSkill.game === game){
            return (gameSkill.skillLevel > 0) ? true : false;
        }
}
return false;
```